benchmarking/generate_l1_svm_lp.jl

```diff
@@ -82,25 +101,82 @@ function parse_command_line()
   return ArgParse.parse_args(arg_parse)
 end
 
+"""
+Loads a LIBSVM file and returns a loaded SvmTrainigData struct.
```

Comment:
nit: "returns a loaded" -> "returns the loaded"

Reply:
Done.